base/complex.jl

```diff
@@ -1125,3 +1125,13 @@ function complex(A::AbstractArray{T}) where T
     end
     convert(AbstractArray{typeof(complex(zero(T)))}, A)
 end
+
+## Double-word complex numbers
+
+const CmplxFlt = Complex{<:AbstractFloat}
```

Comment:
```suggestion
const ComplexFloat = Complex{<:AbstractFloat}
```

And likewise in the rest of the code - source code is for reading after all, I don't think anything is gained by omitting vowels.